code/modules/vending/vending.dm

```diff
@@ -2552,6 +2552,22 @@ TYPEINFO(/obj/machinery/vending/monkey)
 		//product_list += new/datum/data/vending_product(/obj/item/device/timer, rand(1, 6), hidden=1)
 
 
+/obj/machinery/vending/standard/toxins
+	desc = "A vending machine machine full of various useful tools and devices that plasma researchers can use to make bombs."
+	icon_state = "toxins"
+
+	create_products(restocked)
+		product_list += new/datum/data/vending_product(/obj/item/device/prox_sensor, 10)
+		product_list += new/datum/data/vending_product(/obj/item/device/igniter, 10)
+		product_list += new/datum/data/vending_product(/obj/item/device/radio/signaler, 10)
+		product_list += new/datum/data/vending_product(/obj/item/assembly/time_ignite, 3)
```

Comment:
I understand that this provides parity on some maps, but I don't see why this is needed.  Igniter and timer are already available.

Reply:
> I understand that this provides parity on some maps, but I don't see why this (`/obj/item/assembly/time_ignite`) is needed. Igniter and timer are already available.

I felt the same way, but I didn't want to be "removing" items that were previously available  --  It's not just some maps, Atlas, Clarion, cog1, cog2, donut 2, donut 3, kondaru and oshan all had them.  If you're happy with them just being removed outright, I'd prefer that too!

Reply:
On second thought, I just ripped em out.  If someone else decides that they were, like, load-bearing important to the toxins experience they can always PR adding them in.  I personally have never used one of the pre-built assemblies